cli/tests/install.bats

```diff
@@ -454,7 +454,7 @@ EOF
   assert_failure
   assert_output "$(
     cat << EOF
-❌ ERROR: resolution failed: The attr_path python311Packages.torchvision-bin is not found for all requested systems on the same page, consider package groups with the following system groupings: (aarch64-darwin,x86_64-linux), (x86_64-linux), (aarch64-darwin,x86_64-darwin,x86_64-linux), (aarch64-darwin,aarch64-linux,x86_64-linux).
+❌ ERROR: resolution failed: The attr_path python311Packages.torchvision-bin is not found for all requested systems on the same page, consider package groups with the following system groupings: (aarch64-darwin,aarch64-linux,x86_64-linux), (x86_64-linux), (aarch64-darwin,x86_64-linux), (aarch64-darwin,x86_64-darwin,x86_64-linux).
```

Comment:
Filed https://github.com/flox/catalog-server/issues/187 to prevent this in future.